File: Arrays/09/sol.py
for t in range(int(input())):
    size = input()
    arr = [int(x) for x in input().split()]
    ls = [0, 0, 0]
    for i in arr:
        ls[i] += 1
    ans = '0 '*ls[0] + '1 '*ls[1] + '2 '*ls[2]
    print(ans)

File: Arrays/14/sol.java


import java.util.*;
import java.lang.*;
import java.io.*;
class GFG
 {
	public static void main (String[] args)throws Exception
	 {
	    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
	    int test = Integer.parseInt(br.readLine());
	    while (test-- > 0)
	    {
	        int size = Integer.parseInt(br.readLine());
	        int []arr = new int[size];
	        String []buff = br.readLine().split("\\s+");
	        for (int i = 0; i < size; i++)
	        {
	            arr[i] = Integer.parseInt(buff[i]);
	        }
	        int k = Integer.parseInt(br.readLine());
	        Arrays.sort(arr);
	        System.out.println(arr[k-1]);
	    }
	 }
}
